sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -0,0 +1,543 @@
+# Azure Text Analytics client library for JavaScript
+
+[Azure Cognitive Service for Language](https://azure.microsoft.com/services/cognitive-services/language-service/) is a cloud-based service that provides advanced natural language processing over raw text, and includes the following main features:
+
+**Note:** This SDK targets Azure Cognitive Service for Language API version 2022-04-01-preview.
+
+- Language Detection
+- Sentiment Analysis
+- Key Phrase Extraction
+- Named Entity Recognition
+- Recognition of Personally Identifiable Information
+- Linked Entity Recognition
+- Extractive Summarization
+- Healthcare Analysis
+- Custom Entities Recognition
+- Custom Document Classification
+- Support Multiple Actions Per Document
+
+Use the client library to:
+
+- Detect what language input text is written in.
+- Determine what customers think of your brand or topic by analyzing raw text for clues about positive or negative sentiment.
+- Automatically extract key phrases to quickly identify the main points.
+- Identify and categorize entities in your text as people, places, organizations, date/time, quantities, percentages, currencies, healthcare specific, and more.
+- Perform multiple of the above tasks at once.
+
+Key links:
+
+- [Source code](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/textanalytics/ai-text-analytics/)
+- [Package (NPM)](https://www.npmjs.com/package/@azure/ai-text-analytics)
+- [API reference documentation](https://docs.microsoft.com/javascript/api/@azure/ai-text-analytics)
+- [Product documentation](https://docs.microsoft.com/azure/cognitive-services/language-service/)
+- [Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/textanalytics/ai-text-analytics/samples)
+
+## Getting started
+
+### Currently supported environments
+
+- [LTS versions of Node.js](https://nodejs.org/about/releases/)
+- Latest versions of Safari, Chrome, Edge, and Firefox.
+
+See our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
+
+### Prerequisites
+
+- An [Azure subscription][azure_sub].
+- An existing [Cognitive Services][cognitive_resource] or Language resource. If you need to create the resource, you can use the [Azure Portal][azure_portal] or [Azure CLI][azure_cli].
+
+If you use the Azure CLI, replace `<your-resource-group-name>` and `<your-resource-name>` with your own unique names:
+
+```PowerShell
+az cognitiveservices account create --kind TextAnalytics --resource-group <your-resource-group-name> --name <your-resource-name> --sku <your-sku-name> --location <your-location>
```

Comment:
we should double-check that `--kind` here still works with TextAnalytics and/or if we should update this to Language

Reply:
It seems to be working fine for CI resouces: https://github.com/Azure/azure-sdk-for-js/blob/3704fc7f4f9f34576df0476e66b6ea07395d557f/sdk/textanalytics/test-resources.json#L46. However, I agree that we should probably use the new names if any. I tried `Language` but it doesn't work. Also [CLU's `test-resources.json`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/test-resources.json) seems to be still under construction. @heaths do you know what is the kind of Language resources?

Reply:
Sorry, no. It would be best to look at the swaggers for the acceptable values.

Also, our test-resources.json - and for Python, now that we all use the same - isn't under construction. It's that we use static test resources for now. Since training doesn't take too long, eventually we'll want to deploy resources like most SDKs, but it's lower priority right now.

Reply:
@heaths Thanks for sharing your input and thoughts! May I ask where could we find that information in the [swagger](https://github.com/Azure/azure-rest-api-specs/blob/cognitiveservices-Language-2022-04-01-preview/specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json)?